docs/Product-Guides/Request-Forwarding.md

```diff
@@ -0,0 +1,58 @@
+---
+stoplight-id: sl9n3gphljf2w
+---
+
+# Request Forwarding
+
+## Enabling Custom Data Requests
+
+Request Forwarding is a passthrough API feature that enables you to issue raw requests directly against an integration. With Request Forwarding, you have the ability to access any functionality that is natively supported by an integration. You’ll be able to access data elements that are outside of Finch’s existing API structure and standard data model. You’ll also be able to write data elements that are outside the scope of what is supported through one of Finch’s standardized APIs.
+
+*Request Forwarding is available for customers in our Scale tier. Please reach out to a Finch representative for more details on eligibility.*
+
+### How it Works
+
+**Standard Finch API Request**
+
+Requests to one of Finch’s standard API endpoints go through a data transformation layer to ensure that you have a consistent request and response structure to work with, regardless of the system you’re extracting data from.
+
+<!-- focus: false -->
+![finch_standard_api.png](../../assets/images/finch_standard_api.png)
+
+**Finch Request Forwarding**
+
+Request Forwarding bypasses the data transformation layer, giving you access to the raw data exposed by an integration and leaving the data mapping fully in your control.
+
+<!-- focus: false -->
+![finch_request_forwarding.png](../../assets/images/finch_request_forwarding.png)
+
+For more details on making requests using Request Forwarding, please refer to the API reference for `/forward`.
+
+### Using Request Forwarding
+
+You can watch a video demonstration that exhibits how to use the /forward API for [accessing deeper data sets](https://www.loom.com/share/81f86fa4a2fc4c13b42bb153e66dba06?sid=5ba4e1df-840a-40f6-b2f7-4abbb71eb01c) within an employment system.
```

Comment:
i would place the link in "watch a video demonstration" to indicate that the link is to the video more clearly